src/kotlinSessions/SessionThree.kt

```diff
@@ -0,0 +1,231 @@
+package kotlinSessions
+
+/**
+ * The main class is the entry point of your application.
+ */
+
+fun main() {
+    //---------------------------------------Arrays----------------------------------------------
+    // An Array : is a list of some items, the items can be of Any type.
+    // the items have an order starting from (0) to (size - 1) and we usually call it an index.
+    // to get a particular item from an array, we append the subscript notation '[]' to our variable name and write our corresponding index inside
+    // asserting that its within the array range (0, size - 1).
+    // to define an Array use the type Array<Type> which will define an array of a specific type
+    val studentNames: Array<String> = arrayOf("Maged", "Abdo")
+    studentNames.forEach {
+        println(it)
+    }
+    // another way to define arrays of fixed size.
+    // the block {} is called an initialize phase for the constructor, it creates default values for our array while initializing it using the indices
+    val students: Array<String> = Array(3) {
+        index: Int -> "Name"
+    }
+    students.forEach {
+        println(it)
+    }
+    // modifying our array
+    students[0] = "Maged"
+    students[1] = "Mena"
+    students[2] = "Morad"
+    students.forEach {
+        println(it)
+    }
+    // print our size
+    println("Array Size = ${students.size}")
+
+    //printing our items using ranges
+    for (index in students.indices step 1) {
+        println(students[index])
+    }
```

Comment:
no need for step 1 because it is the init value

Reply:
I just want to remind them that we can control the loop steps using the step function.